uint8 agent_number
 WaypointTask waypoint_list
